packages/components/src/theme/ComposerTheme/Components/Radio.ts

```diff
@@ -0,0 +1,71 @@
+import { radioAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import { PartsStyleInterpolation } from '@chakra-ui/theme-tools';
+
+const sizes: Record<string, PartsStyleInterpolation<typeof parts>> = {
+  sm: ({ theme }) => ({
+    ...theme.textStyles['text-body-meta-regular'],
+  }),
+  md: ({ theme }) => ({
+    ...theme.textStyles['text-body-regular'],
+  }),
+  lg: ({ theme }) => ({
+    ...theme.textStyles['text-body-large-regular'],
+  }),
+};
+
+export const Radio: ComponentMultiStyleConfig = {
+  parts: parts.keys,
```

Comment:
Is this needed?

Reply:
yes. it was necessary because I was using `ComponentMultiStyleConfig` type that requires parts prop but it existed a union type `ComponentStyleConfig` which join `ComponentMultiStyleConfig` and `ComponentSingleStyleConfig` and in that case that field is not needed. 

just to add that following ChakraUi documentation using parts props is recommended when you are working with [MultiPart components ](https://chakra-ui.com/docs/theming/component-style#styling-multipart-components) I think it makes more readable the component.